test/ANNOTATIONS.yaml

```diff
@@ -670,6 +678,8 @@ mg-b:
     - Move chpl_getPrivatizedClass() into chpl-privatization.h (#6212)
   05/23/17:
     - Fix NPB MG (#6284)
+  06/13/17:
+    - Optimize updateFluff (#6418)
```

Comment:
This impacted the prk stencil as well. @benharsh any others that you would expect to be impacted?